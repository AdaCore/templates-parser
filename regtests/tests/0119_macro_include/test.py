from test_support import *

run('print_tree', ["main.tmplt"])
run('testme', ["main.tmplt"])
