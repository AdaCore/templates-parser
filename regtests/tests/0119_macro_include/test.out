[INFO] main.tmplt
 -> mincl.incl
[C_INFO]  1 FALSE
[TEXT] Some text:
[TEXT] @_WINCL(A,B)_@
[TEXT] --
[TEXT] @_WINCL(98,123)_@
Some text:
before
include B and A
Another A B
after
--
before
include 123 and 98
Another 98 123
after
